### ISSUE TITLE
Mongo driver 1.6.0 not working with PHP 7.4 TS x64

### ISSUE THREAD

**Issue by globules-io** [mongodb/mongo-php-driver#1070]
Uncaught Error: Class 'MongoDB\Driver\Manager' not found
This is what we get with PHP7.4 x64 TS + php_mongodb-1.6.0-7.3-ts-vc15-x64
Your DLL not does work on x64

**Comment by jmikola** [comment]
https://pecl.php.net/package/mongodb/1.6.0/windows does not have any DLLs for PHP 7.4, as that version of the driver was released before PHP 7.4. It is expected that a DLL compiled for PHP 7.3 would not work with 7.4, as the PHP version, architecture, and thread-safety are all relevant.

PECL is responsible for producing Windows builds. I'm not sure if the administrators went back and attempted to compile old extension releases for PHP 7.4, but it wouldn't have worked for the MongoDB driver due to [PHPC-1503](https://jira.mongodb.org/browse/PHPC-1503) anyway. We released [1.6.1](https://github.com/mongodb/mongo-php-driver/releases/tag/1.6.1) earlier today with that fix, so I expect DLLs should show up on https://pecl.php.net/package/mongodb/1.6.1/windows within a few hours.

Feel free to close this ticket if you see them appear later today; otherwise, we'll follow up and close this once we notice them.
